packages/shared/src/components/profile/ProfileSettingsMenu.tsx

```diff
@@ -72,6 +74,16 @@ const useMenuItems = (): NavItemProps[] => {
         }
       : undefined;
 
+    const downloadAndroidApp =
+      checkIsBrowser(UserAgent.Android) && !isAndroidApp()
+        ? {
+            label: 'Download mobile app',
+            icon: <GooglePlayIcon />,
+            href: process.env.NEXT_PUBLIC_ANDROID_APP,
+            target: '_blank',
```

Comment:
Do we auto add `rel` if this set? (inside the render, else we'll need to pass it)